frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.unit.spec.js

```diff
@@ -427,4 +459,42 @@ describe("FieldValuesWidget", () => {
       expect(fetchFieldValues).not.toHaveBeenCalledWith(EXPRESSION_FIELD_ID);
     });
   });
+
+  describe("NoMatchState", () => {
```

Comment:
I checked out the branch locally and reverted changes to `FieldValuesWidget` and this test still passes. I don't think it verifies changes in this PR.